### ISSUE TITLE
nlog-project.org - feed not well-formed

### ISSUE THREAD

**Issue by snakefoot** [NLog/NLog#3495]
https://nlog-project.org/feed/

![image](https://user-images.githubusercontent.com/11509660/59965270-0bb3f500-950c-11e9-87ae-614ce935e641.png)

**Comment by 304NotModified** [comment]
lol I guess that's ages already.

anyway, `&` -> ~&amp;`

**Comment by snakefoot** [comment]
Chrome has another error:

![image](https://user-images.githubusercontent.com/11509660/59968083-d6bb9880-9533-11e9-86e1-7c2010a59157.png)

**Comment by snakefoot** [comment]
And the forbidden IE11 says this:

![image](https://user-images.githubusercontent.com/11509660/59968093-09659100-9534-11e9-9c6a-562baa328edf.png)

**Comment by snakefoot** [comment]
Guess they are all say the same. But Firefox gives the best reason :)

**Comment by 304NotModified** [comment]
Yes, it's invalid XML. Dunno how the feed is exactly constructed. Have to check that
